src/Database/Runes/Collection.ts

```diff
@@ -0,0 +1,20 @@
+import { BlockInfo, RuneEtching, RuneUtxoBalance } from "runestone-lib";
+
+import { CollectionRegistrar, mongo } from "~Services/Mongo";
+
+export const collectionBlocks = mongo.db.collection<BlockInfo>("runes_blocks");
+export const collectionUtxoBalances = mongo.db.collection<RuneUtxoBalance>("runes_utxoBalances");
+export const collectionEtchings = mongo.db.collection<RuneEtching>("runes_etchings");
+
+export const registrarBlocks: CollectionRegistrar = {
```

Comment:
What do we need to store the blocks for? Seems like we're storing the block time and previous block hash. If this is not frequently queried, we can just pull the data from the rpc and not store it on the DB.

Reply:
It is used internally by RuneUpdater (runestones-lib). Using it for simplicity and maintainability for now. The collection is not very heavy (20Mb fully indexed in signet) and since its not storing txs its not going to grow much bigger in other networks.